stage0/app.go

```diff
@@ -459,12 +462,22 @@ func removeAppFromPodManifest(pm *schema.PodManifest, appName *types.ACName) {
 }
 
 func StartApp(cfg StartConfig) error {
-	p, err := stage1types.LoadPod(cfg.Dir, cfg.UUID)
+	pod, err := pkgPod.PodFromUUIDString(cfg.DataDir, cfg.UUID.String())
 	if err != nil {
-		return errwrap.Wrap(errors.New("error loading pod manifest"), err)
+		return errwrap.Wrap(errors.New("error loading pod"), err)
 	}
+	defer pod.Close()
 
-	pm := p.Manifest
+	debug("locking pod manifest")
+	if err := pod.ExclusiveManifestLock(); err != nil {
```

Comment:
wait, why do we need to lock the manifest during `app start`/`app stop`. We are not mutating here anything and updates on the pod manifest are atomic.